opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/ais/TransactionListingService.java

```diff
@@ -30,7 +30,8 @@
 
     @Override
     protected void doValidate(DelegateExecution execution, TransactionListXs2aContext context) {
-        validator.validate(execution, extractor.forValidation(context));
+        context.setClassName(this.getClass().getName());
```

Comment:
It is better to pass Class (not name) directly to validator

Reply:
ok